pages/mixed-content.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: Mixed Content
+permalink: /mixed-content/
+---
+
+### What is mixed content?
+
+When you view a page in your browser, it is almost never just a single request. Almost every web page contains resources, such as Javascript, CSS, and images, which cause additional requests. Sometimes these requests are to the same servers as the rest of the site, other times they are external resources (such as Google Analytics snippets of JavaScript).
+
+When the main page is loaded over `HTTPS` and these sub-resources are loaded over `HTTP` this is called **mixed content**.
```

Comment:
Not sure who the audience is, but the spec is meant to be pretty readable (and might lend credence to the notion that this is a Standard Thing That Browsers Do That You Need To Care About™): http://www.w3.org/TR/mixed-content/